### ISSUE TITLE
Kytos start and stop

### ISSUE THREAD

**Issue by beraldoleal** [kytos/kytos-utils#58]
Systemd init scripts should be able to execute this, but besides that we also should be able to start / stop / restart / status controller via kytos command line:

```
$ kytos controller stop
$ kytos controller start
```

**Comment by cemsbr** [comment]
Depends on #57

**Comment by hdiogenes** [comment]
> Depends on #57 [implement a `kytos connect` command to remotely open kytos console]

Not exactly. We could implement the `stop` feature through the REST API, but the `start` command by definition won't work that way. @beraldoleal, we need to review what was the objective here.
